doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -59,6 +59,8 @@
 - MinGW support for Windows [#2150](https://github.com/eclipse-iceoryx/iceoryx/issues/2150)
 - Add support for `iox::string` in `UnixDomainSocket` and created `unix_domain_socket.inl` [#2040](https://github.com/eclipse-iceoryx/iceoryx/issues/2040)
 - Add support for `iox::string` in `MessageQueue` and created `message_queue.inl` [#1963](https://github.com/eclipse-iceoryx/iceoryx/issues/1963)
+- Add support for `iox::string` in `NamedPipe` and created `named_pipe.inl` [#1963](https://github.com/eclipse-iceoryx/iceoryx/issues/1963)
```

Comment:
Oh no. I did not pay attention to the issue number on the last PR. You mixed two digits up. It's 1693 instead of 1963.

Can you please close this PR and create one where the branch name has a 1693 instead of a 1963.
In that PR please also do a
```
git revert 890848f09
git cherry-pick 890848f09
```
and change the commit message of the cherry-picked commit to `iox-#1693 ...`

Reply:
On a second thought, I'll do the revert and cherry-pick on a branch where I'm currently replacing the `IOX_EXPECTS` with the new error handling macros. This will prevent merge conflicts later on. Please just create a new PR with a branch name with the correct issue number.

Reply:
Ok, now should be fixed. The branch name was correct, the issue was only in ```iceoryx-unreleases.md``` and in the commit, so I force pushed in this breach!

Reply:
So, now the issue is in [this](https://github.com/eclipse-iceoryx/iceoryx/commit/641d81d534936565898e1f9a81e70b70a9754c56) commit on the master branch, which has in ```iceoryx-unreleases.md``` and in the commit message the wrong issue number (sorry for that).
Tell me how to proceed!

Reply:
No Problem, I fixed it in #2181. It was just two git commands and a change of the commit message. Let's see if we can get this in before #2181 so that you don't have to fix the merge conflicts :)